packages/yourloops/test/integration/mock/data.api.mock.ts

```diff
@@ -116,6 +128,30 @@ export const completeDailyViewData: Data = {
     { time: '2022-08-08T08:00:00Z', type: 'deviceEvent', id: PARAMETER_ID, lastUpdateDate: '2022-08-08T08:00:00Z', level: '1', name: 'MEAL_RATIO_LUNCH_FACTOR', previousValue: '110', subType: 'deviceParameter', timezone: 'UTC', units: '%', uploadId: 'osef', value: '100', _userId: 'osef' }
   ]
 }
+export const emptyWeightData: Data = {
```

Comment:
This variable name is really misleading.

- This is a big ass structure but the name suggests that it is just to have data with an empty weight
- It is used in totally unrelated places such as `should be able to manage medical reports` or `should render correct components when navigating to a patient scoped on the private team`

What does this data really represents? Do you need all of this?